app/routes.js

```diff
@@ -12,6 +12,14 @@ const offencesList = require('./data/offence_list');
 const https = require('https');
 const utils = require('./utils');
 
+router.use((req, res, next) => {
+  let mvpFlag = req.query && req.query.mvp;
```

Comment:
Something Ruby-y that looks like it's been added to js now - Optional Chaining - I think would work and be more consise here
`req?.query?.mvp`

Reply:
https://stackoverflow.com/questions/32139078/null-safe-property-access-and-conditional-assignment-in-es6-2015